source/va.sqf

```diff
@@ -0,0 +1 @@
+["Open",true] call BIS_fnc_arsenal;
```

Comment:
I know this is nit-picky but could we come up with a more descriptive file name than `va.sqf`? Maybe `bisArsenal.sqf` or something. I'm really open to suggestions here. I just really hate acronym variables. Most worth-while editors will also allow for auto-complete so I feel the "too much typing" argument is a bit weak especially when the name is descriptive of what the item does.